### ISSUE TITLE
Idea: add option to remove the inner jetties of skeletonized baseplates along the longer side

### ISSUE THREAD

**Issue by LorenzKahl** [kennetek/gridfinity-rebuilt-openscad#47]
If checked the marked areas would be dropped.
![4C4EE15C-6096-49C1-9048-2E2C53C5C0A7_1_201_a](https://user-images.githubusercontent.com/301135/211430737-ddd60686-3c3f-4f39-956e-99449834e0ae.jpeg)

This could save some filament without impacting stability a lot - I guess.
Unfortunately I'm a total OpenSCAD NOOB so unfortunately I can't chip in a PR for this. I also assume that the required logic for this is not quiet that easy.

**Comment by Ruudjhuu** [comment]
Interesting idea, however, I do think it will impact stability a lot if the baseplate gets bigger. I created a POC which only works in the x direction for anyone to try.

[poc/inner_jeties branch in Ruudjhuu/gridfinity-rebuilt-openscad](/Ruudjhuu/gridfinity-rebuilt-openscad/tree/poc/inner_jeties)

**Comment by LorenzKahl** [comment]
Hi @Ruudjhuu, thank you, but the link seems to be dead. Could you provide another source and I'll be trying it out.

**Comment by Ruudjhuu** [comment]
The link is working for me, Maybe this one works?

[https://github.com/Ruudjhuu/gridfinity-rebuilt-openscad/tree/poc/inner_jeties](https://github.com/Ruudjhuu/gridfinity-rebuilt-openscad/tree/poc/inner_jeties)

**Comment by LorenzKahl** [comment]
Yeah, you were right. I was on my phone earlier and when I clicked the link it was directing me to `localhost`.
The model looks fine to me. I'm starting a print right now and will report tomorrow how it came out. The savings of filament seem to be a little less than 10% which is quiet nice ;-)

Unfortunately I can't go any bigger than 4x4 on my Mini.
<img width="1545" alt="Bildschirm­foto 2023-01-18 um 23 20 58" src="https://user-images.githubusercontent.com/301135/213309116-897a74fb-94c7-419b-8630-e3e2b8a7d2ee.png">
<img width="1484" alt="Bildschirm­foto 2023-01-18 um 23 21 45" src="https://user-images.githubusercontent.com/301135/213309203-c0ce017f-e993-40f6-a9cd-6c9e21501321.png">

**Comment by LorenzKahl** [comment]
Hi @Ruudjhuu, the PoC came out really nice! I added some pictures and videos to get an impression. 
Off course, the stiffness of the whole thing reduces the bigger your prints get. But for tucking the baseplate in a drawer I don't see any problem. Also the asymmetric look kinda bothers my eye, but again - you won't even see it when it's stuffed in a drawer and stacked with bins.

I would really like to see this added to the [gridfinity-rebuilt-openscad](https://github.com/kennetek/gridfinity-rebuilt-openscad) repo 😍 

Thanks for giving this a shot!
![IMG_1500](https://user-images.githubusercontent.com/301135/213390258-4430512b-667b-4fb5-b6cf-b1cd2c82ef22.jpeg)

Videos on YT:
https://youtu.be/gqlihwKOFuw
https://youtu.be/reW4JjOzt7s

**Comment by kennetek** [comment]
@Ruudjhuu I'd recommend adding this as another style to go with your other minimalist baseplates.
